types/jquery.validation/index.d.ts

```diff
@@ -17,6 +17,13 @@ declare namespace JQueryValidation {
          * default: false
          */
         debug?: boolean | undefined;
+        /**
+         * Use this to register additional elements to be validatable. Main use case is for validating web components.
+         * By default, only the following elements are supported: "input", "select", "textarea", "[contenteditable]"
+         * 
+         * default: []
+         */
+        customElements?: string[] | undefined;
```

Comment:
Interestingly this is present in the source code, but not in the documentation.

Reply:
Yes, true. I just opened a PR to fix this in the documentation as well https://github.com/jquery-validation/validation-content/pull/59. I'll wait for an approval there and will merge this PR once the documentation PR is merged. Thanks for your review! ❤️